src/main/java/com/societegenerale/commons/plugin/model/ScopePaths.java

```diff
@@ -0,0 +1,25 @@
+package com.societegenerale.commons.plugin.model;
+
+/**
+ * Can be used in the Maven or Gradle plugins that use the core, to override the scope paths in the configuration
+ */
+public class ScopePaths {
```

Comment:
To reduce the verbosity , how about using [record](https://docs.oracle.com/en/java/javase/17/language/records.html) ?

Reply:
I don't remember what this is about.. open for so long, the parent issue is closed... I'll close it.